nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -125,11 +136,16 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
         }
 
         override fun onCharacteristicRead(gatt: BluetoothGatt, characteristic: BluetoothGattCharacteristic, status: Int) {
+            writeQueue.next()
+
             if (characteristic.uuid.equals(BATTERY_CHARACTERISTIC_UUID)) {
-                matrixWriter?.onWrite()
```

Comment:
1) It needs to happen for all cases, not just battery characteristic
2) It wasn't the proper method to call here. writeQueue.next() should have been called. It only worked because onWrite
() also called writeQueue.next()